velox/docs/monitoring/stats.rst

```diff
@@ -18,24 +18,50 @@ analysis.
 
 Memory Arbitration
 ------------------
+These stats are reported by all operators.
+
 .. list-table::
    :widths: 40 10 50
    :header-rows: 1
 
    * - Stats Name
-     - Type
+     - Unit
      - Description
    * - memoryReclaimCount
-     - kNone
+     - <empty>
      - The number of times that the memory arbitration to reclaim memory from
        an spillable operator.
        This stats only applies for spillable operators.
    * - memoryReclaimWallNanos
-     - kNano
+     - kNanos
```

Comment:
kNanos, kBytes are not English words. Let's use "nanos" and "bytes" without quotes.

Reply:
Fixed